python/fledge/common/configuration_manager.py

```diff
@@ -301,6 +301,19 @@ def get_entry_val(k):
                         else:
                             d = {entry_name: entry_val}
                             expected_item_entries.update(d)
+                    elif entry_name == "permission":
+                        if not isinstance(entry_val, list):
+                            raise ValueError(
+                                'For {} category, {} entry value must be in list for item name {}; got {}.'
```

Comment:
in list -> a list of user roles

Reply:
Here we are checking instance type only

Reply:
in is not correct, `a list` even better `a list of string`